css/styles.css

```diff
@@ -0,0 +1,105 @@
+@import "https://fonts.googleapis.com/css2?family=Asap:wght@100;300;400;500;600&display=swap";
+
+* {
+  box-sizing: border-box;
+  background-color: #f0f0f0;
+  font-family: Asap, sans-serif;
+}
+
+.header-title {
+  background-color: #552b80;
+  color: #f0f0f0;
+  padding: 1.2rem 7.2rem;
+  margin: 0 -7.5rem;
+  font-size: 1.8rem;
+  font-weight: 300;
+}
+
+.new-entry {
+  font-size: 2.8rem;
+  margin: 1.3rem 0;
+  font-weight: 500;
+}
+
+.input,
+.input-notes {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.placeholder-image {
+  object-fit: cover;
+  width: 100%;
+  height: 320px;
+  margin-bottom: 1rem;
+}
+
+input,
+textarea {
+  margin: 0.8rem 0;
+  border-radius: 2%;
+  border: 1px solid #cfcfcf;
+  background-color: white;
+}
+
+label {
+  font-size: 1.3rem;
+  font-weight: 500;
+}
+
+.notes {
+  height: 200px;
+}
+
+.title,
+.photo {
+  height: 40px;
+}
+
+.save-button {
+  background-color: #552b80;
+  color: #f0f0f0;
+  margin: 1rem 0 1.5rem;
+  border: none;
+  border-radius: 5%;
+  padding: 0.5rem 1rem;
+  font-size: 1.2rem;
+  letter-spacing: 1px;
+}
+
+.button {
+  text-align: right;
+}
+
+@media only screen and (min-width: 768px) {
+  .container {
+    width: 100%;
+  }
+
+  .header-title {
+    margin: 0 -20.75rem;
+    padding-left: 325px;
```

Comment:
You should not be using a set `px` value for padding, because this won't dynamically update according to the screen size